tests/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(6, 2)) == 6
 
 
 def test_should_split_a_number_into_equal_parts_when_value_is_divisible_by_number_of_parts():
-    pass
+    goals = split_integer(20, 4)
+
+    for item in goals[:-1]:
+        assert item == 20 / 4
 
 
 def test_should_return_a_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    goals = split_integer(18, 4)
+
+    assert goals[0] == 4
```

Comment:
This test should cover the case when you divide an integer not by four, but by one part

Reply:
Sorry, i can not understand. Do i need to divide 18 by 1, like split_integer(18, 1) ?

Reply:
Right 👍

Reply:
Changes provided. Thanks for explanation, the test function name was not so clear for me.